scss/detail.scss

```diff
@@ -33,13 +33,21 @@
   width: 540px;
 }
 
-.book-container {
+.main-container {
+  box-shadow: 4px 4px 10px 0 $black-05;
   display: flex;
+  height: 416px;
   margin: 24px;
+  width: 900px;
```

Comment:
Are you sure you need the height and width?, at this point book container is taking its content space. Also with the box-shadow in book container you will have to change the margin with padding.